junit-platform-commons/src/main/java/org/junit/platform/commons/util/ClasspathResource.java

```diff
@@ -10,20 +10,30 @@
 
 package org.junit.platform.commons.util;
```

Comment:
Should we move this class to the `org.junit.platform.commons.support` where the `Resource` interface resides and call it `DefaultResource`?